data/json/snippets.json

```diff
@@ -2292,6 +2292,34 @@
       "XE037 revivification has reached critical levels, which are rapidly overwhelming the available military and regional police assets' ability to stop.  Every team in the field has encountered hostile specimens, and several are no longer responding to communications.  Dr. Savage has proposed a strategic redeployment to the underground complex known as the VAULT, and continuing production of PE062 there."
     ]
   },
+  {
+    "type": "snippet",
+    "category": "radio_archive",
+    "text": [ 
+      "kssht.  Dark Horse, this is Blue Jay, what's your status, over.  kssht.  Blue Jay, this is Black Horse, still holding, but not for long.  kssht.  Dark Horse, you've got to hold position for 3 hours.  We're almost black on ammo, but resupply is on the way, over.  kssht.  Not possible Blue Jay, too many dead. 30 minutes max, and if you don't order us to retreat we're gone, over.  kssht.",
```

Comment:
```suggestion
      "kssht.  Dark Horse, this is Blue Jay, what's your status, over.  kssht.  Blue Jay, this is Black Horse, still holding, but not for long.  kssht.  Dark Horse, you've got to hold position for 3 hours.  We're almost black on ammo, but resupply is on the way, over.  kssht.  Not possible Blue Jay, too many dead.  30 minutes max, and if you don't order us to retreat we're gone, over.  kssht.",
```

Reply:
Blue Jay is calling **Dark** Horse, but it's **Black** Horse coming back.